aten/src/ATen/OpaqueTensorImpl.h

```diff
@@ -80,13 +80,14 @@ struct CAFFE2_API OpaqueTensorImpl : public TensorImpl {
 // 1. the AutogradMeta pointer, because it is unique for each Variable.
 // 2. the version counter, because although it lives in TensorImpl, the version counter is managed
 // by autograd, and the call sites of `shallow_copy_and_detach()` (from autograd) should decide what
-// the version counter should be for each new TensorImpl. See NOTE [ Version Counter Sharing ] for details.
+// the version counter should be for each new TensorImpl, by passing the correct version counter as
```

Comment:
I don't know what "the version counter should be for each new TensorImpl" is supposed to mean.

Reply:
I still don't know what this means.  Can you just say something like "the version counter is set to the passed in`version_counter`.